azerothcore/azerothcore-wotlk
# feat(DB/Account): Delete account_muted when deleting an account

## CHANGES PROPOSED:
-  Remove the data account_muted in account deletion
-  It'll make the database cleaner.

## ISSUES ADDRESSED:


## TESTS PERFORMED:
- Tested in the game
-

## HOW TO TEST THE CHANGES:
Use the `.mute` command on any player then delete their  `.account delete $account`
you'll see that the data is not stored.


## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
